documentloaders/notion.go

```diff
@@ -0,0 +1,57 @@
+package documentloaders
```

Comment:
please include a doc.go in new packages.

Reply:
in python codebase, notion directory loader belongs to document loaders so i thought i shouldn't create a new packafe and have it as part of documentloaders. documentloaders has already have a doc.go so i didn't include new one. i'm also working on notiondb loader btw